flasher_stub/README.md

```diff
@@ -0,0 +1,3 @@
+This is the source of CesantaFlasher.FLASHER_STUB.
+
+Environment to compile it is available [here](https://github.com/cesanta/fnc/tree/master/common/platforms/esp8266/stubs).
```

Comment:
Do you think it would be viable to include enough additional files to compile the stub in-tree here? (Except for the 3 small SDK header files, at least.)